.appveyor.yml

```diff
@@ -0,0 +1,9 @@
+version: 'Build-{build}'
```

Comment:
This is the name of the build, where `{build}` is the incremental number.

I think that we should change it to something better, `Build-1` and `Build-2` don't seem to sound very well.

Reply:
`{build}` is required.

What about `{build}-{commit-sha1}`?